src/applications/vaos/newAppointmentFlow.js

```diff
@@ -78,6 +84,8 @@ export default {
 
       if (isSleepCare(state)) {
         nextState = 'typeOfSleepCare';
+      } else if (isEyeCare(state)) {
```

Comment:
I can't highlight this very well, but can you move line 135 in this file (`dispatch(updateFacilityType(FACILITY_TYPES.VAMC));`) and put it above line 86? 

We're selecting VAMC on the type of facility page inadvertently when you get to it through the eye care page. Sleep care is the only branch where it needs to be set now.

Reply:
Done